app/gateway/2.7.x/developer-portal/using-the-editor.md

```diff
@@ -18,54 +16,36 @@ Kong Manager offers a robust file editor for editing the template files of the D
 
 From the **Kong Manager** dashboard of your **Workspace**, click **Editor** under **Dev Portal** in the sidebar.
 
-![Launch Editor Mode](https://doc-assets.konghq.com/1.3/dev-portal/editor/devportal-editor-sidebar-button.png)
-
-
-This will launch the **Editor Mode**:
-
-![Editor Mode](https://doc-assets.konghq.com/1.3/dev-portal/editor/devportal-editor-mode-launch.png)
-
-
+This will launch the **Editor Mode**.
 ## Navigating the Editor
 
 When enabled, the Dev Portal is pre-populated with Kong's default theme. The file editor exposes these files to the UI, allowing them to be edited quickly and easily from inside the browser. When you first open the editor, you will be presented with a list of files on the left, and a blank editing form.
 
 1. Create new files for the Dev Portal right from the Editor by clicking `New File+`.
-2. List of all exposed template files in the Dev Portal, separated by Content / Spec / Themes.
-3. Code View - Select a file from the sidebar to show the code here.
-4. Portal Preview - View a live preview of the selected Dev Portal file.
-5. Toggle View - Choose between three different views: full screen code, split view, and full screen preview mode.
+1. List of all exposed template files in the Dev Portal, separated by Content / Spec / Themes.
+1. Code View - Select a file from the sidebar to show the code here.
+1. Portal Preview - View a live preview of the selected Dev Portal file.
+1. Toggle View - Choose between three different views: full screen code, split view, and full screen preview mode.
 
-![Dev Portal with Numbers](https://doc-assets.konghq.com/1.3/dev-portal/editor/devportal-editor-numbers.png)
+![Dev Portal with Numbers](/assets/images/docs/dev-portal/editor-mode-numbered.png)
```

Comment:
Looks like this file is missing: 
https://deploy-preview-3639--kongdocs.netlify.app/gateway/2.7.x/developer-portal/using-the-editor/